core/main/src/service/apps/delegated_launcher_handler.rs

```diff
@@ -146,6 +146,14 @@ impl AppManagerState {
             HashMap::new()
         }
     }
+    /*
+    Adding to suppress:
+    ```note: joining a path starting with separator will replace the path instead
+    help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#join_absolute_paths
+    note: `-D clippy::join-absolute-paths` implied by `-D warnings```
+    to avoid breaking any existing functionality
+    */
+    #[allow(clippy::join_absolute_paths)]
```

Comment:
Don't we just want to remove the `/` below in:
`std::path::Path::new(&env::temp_dir().display().to_string()).join("/app_info")`?

Reply:
Maybe? I assume all these things are they way the are for a good reason, so did not want to arbitrarily change it.

Reply:
@Karthick-Somasundaresan : Was the `/` intentional for some reason?

Reply:
No clue. The only reason this is popping up now is because clippy has become smarter between 1.69.0 and 1.76.0 . I did contemplate adding a unit test(s) , but that seems out of scope for this exercise

Reply:
so.. is this actionable? fix it? leave it? would like to unblock the compiler upgrade if possible

Reply:
I'm pretty sure line 167 is a bug because it will create /app_info and not <temp_dir>/app_info, I vote to just remove the `/` and remove `#[allow(clippy::join_absolute_paths)]`.

Reply:
Thanks @pahearn73 ! Done